cli/cliutils/cliutils.go

```diff
@@ -214,6 +215,49 @@ func GetDockerAuth(domain string) (auth dockerclient.AuthConfiguration, err erro
 		}
 	}
 
+	// try to load cred from configured credential store
```

Comment:
Please put a comment on the GetDockerAuth function indicating that it also obtains creds from a docker credential store if it's in use.